translator/translate/logs/util/filter_retention.go

```diff
@@ -0,0 +1,40 @@
+package util
```

Comment:
This is a nitpick, but I think that the name of the file is misleading. I think it would make sense to use a more directed verb, like `translator/translate/logs/util/validate_retention.go`